generators/generators-commons/src/main/java/uk/gov/justice/services/generators/commons/helper/Names.java

```diff
@@ -125,6 +127,20 @@ public static String nameFrom(final MimeType mimeType) {
         return section.substring(0, section.indexOf('+'));
     }
 
+
+    /**
+     * Construct comma separated list of command/event names
```

Comment:
Is this comma separated or dependent on the delimiter? - comments are not so clear in reflecting the 2nd parameter's impact on the method.